spellbook/models/balances/ethereum/erc721/balances_ethereum_erc721_day.sql

```diff
@@ -13,22 +13,18 @@ with
             ) as day
     )
 
-, daily_balances as
- (SELECT
-    wallet_address,
-    token_address,
-    tokenId,
-    day,
-    lead(day, 1, now()) OVER (PARTITION BY token_address, wallet_address ORDER BY day) AS next_day
-    FROM {{ ref('transfers_ethereum_erc721_rolling_day') }})
-
-SELECT distinct
+SELECT
     'ethereum' as blockchain,
     d.day,
     b.wallet_address,
     b.token_address,
     b.tokenId,
-    nft_tokens.name as collection
-FROM daily_balances b
-INNER JOIN days d ON b.day <= d.day AND d.day < b.next_day
-LEFT JOIN {{ ref('tokens_ethereum_nft') }} nft_tokens ON nft_tokens.contract_address = b.token_address
+    b.num_tokens,
+    nft_tokens.name as collection,
+    nft_tokens.category
+FROM days d
+INNER JOIN {{ ref('transfers_ethereum_erc721_rolling') }} b ON (b.evt_block_time <= d.day AND d.day < b.next_evt)
```

Comment:
minor feedback: for readability, even though with inner join the results are the same, can we switch the FROM to be on {{ ref('transfers_ethereum_erc721_rolling') }} and the JOIN on days? as an outsider looking in for first time, seems that would flow better

Reply:
we are projecting state on days here, makes more sense to me to do it this way around 🤷 

I do not object to changing this around, your call.

Reply:
i certainly don't feel strongly about a change, so i'm good with as-is 👍